### ISSUE TITLE
multiline keys

### ISSUE THREAD

**Issue by jerzybrzoska** [dongyx/hardv#3]
When I run `hardv python.fc` I get the following error `python.fc, line 19: key too large`. My file for learning, python.fc, is:

```
%% This is a card set for hardv(1)
%% Created at Feb 1, 1997
NEXT	2023-09-06 10:11:24 +0200
PREV	2023-09-06 10:11:24 +0200
Q	Rewrite the following with list comprehension:\nsquares = []\nfor x in range(10):\n  squares.append(x**2)
A	squares = [x**2 for x in range(10)]
%%
%%
%%
NEXT	2023-09-06 10:11:24 +0200
PREV	2023-09-06 10:11:24 +0200
Q	This line is longer than just 8 chars, it works, though #define KEYSZ	8.
A	It works
%%
%%
NEXT	2023-09-06 10:11:24 +0200
PREV	2023-09-06 10:11:24 +0200
Q	Rewrite the following with list comprehension:\nsquares = []\nfor x in range(10):\n  squares.append(x**2)
squares = []
for x in range(10):
  squares.append(x**2)
A	squares = [x**2 for x in range(10)]

```
I noticed in `hardv.c` that `KEYSZ` is set to 8. Can your code be tweaked so that the key of a card is displayed
with `\n` turned on (as for the first card in `python.fc` above)?  Or in any other way so that `hardv` support multiline
keys and values.

**Comment by dongyx** [comment]
HardV supports multiline values. Subsequent lines must be indented by a tab character.

	Q	Rewrite the following with list comprehension:
		squares = []
		for x in range(10):
			squares.append(x**2)
	A	squares = [x**2 for x in range(10)]

**Comment by dongyx** [comment]
Closed #3 as completed.
